content/quickstart/node/tracing.md

```diff
@@ -0,0 +1,697 @@
+---
+title: "Tracing"
+date: 2018-07-22T20:29:06-07:00
+draft: false
+class: "shadowed-image lightbox"
+---
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Getting started](#getting-started)
+- [Enable Tracing](#enable-tracing)
+    - [Import Packages](#import-tracing-packages)
+    - [Instrumentation](#instrument-tracing)
+- [Exporting to Stackdriver](#exporting-to-stackdriver)
+    - [Import Packages](#import-exporting-packages)
+    - [Create the Exporter](#create-the-exporter)
+- [Optional: Add a Delay](#optional-add-a-delay)
+- [Viewing your Traces on Stackdriver](#viewing-your-traces-on-stackdriver)
+
+In this quickstart, we’ll gleam insights from code segments and learn how to:
+
+1. Trace the code using [OpenCensus Tracing](/core-concepts/tracing)
+2. Register and enable an exporter for a [backend](/core-concepts/exporters/#supported-backends) of our choice
+3. View traces on the backend of our choice
+
+## Requirements
+- Node
+- Google Cloud Platform account and project
+- Google Stackdriver Tracing enabled on your project
+
+{{% notice tip %}}
+For assistance setting up Stackdriver, [Click here](/codelabs/stackdriver) for a guided codelab.
+{{% /notice %}}
+
+## Installation
+We will first create our project directory, initialize our project, and bootstrap our entry file.
+
+```bash
+mkdir repl-app
+cd repl-app
+
+npm init -y
+
+touch index.js
+```
+
+Put this in your newly generated `index.js` file:
+
+```js
+const stdin = process.openStdin();
+
+function readEvaluateProcessLine(input) {
+  const text = readLine(input);
+  const upperCase = processLine(text);
+
+  console.log(upperCase);
```

Comment:
So the intro shows
```shell
> foo
< FOO
```
but the code would plainly print
```shell
foo
FOO
```
Perhaps please update the intro or better let's print without newlines using `process.stdout.write` e.g. for the prompt
```javascript
process.stdout.write('> ');
const text = readLine(input);
const upperCase = processLine(text);
process.stdout.write('\n< ' + upperCase + '\n\n');
```

Reply:
Thank you for the help, CLI wizard!